### PR TITLE
Feature/timestamp selection

### PR DESCRIPTION
**Pull request by alexmillane**
At Mina's request I have added the option to timestamp vrpn messages with ros time. Additionally I have attempted make more clear the hackery we do to correct for (suspected) time zone differences between local and the vicon tracker machines.

### CODE REVIEW

**Comment by alexmillane** [comment]
@helenol @burrimi

File: src/ros_vrpn_client.cpp [hunk 1]
```diff
@@ -293,29 +333,48 @@ int main(int argc, char* argv[])
   std::string vrpn_server_ip;
   int vrpn_port;
   std::string trackedObjectName;
+  std::string timestamping_system_string;
 
   private_nh.param<std::string>("vrpn_server_ip", vrpn_server_ip, std::string());
   private_nh.param<int>("vrpn_port", vrpn_port, 3883);
   private_nh.param<std::string>("vrpn_coordinate_system", coordinate_system_string, "vicon");
   private_nh.param<std::string>("object_name", object_name, "auk");
+  private_nh.param<std::string>("timestamping_system", timestamping_system_string, "blah");
```
Comment (helenol) [inline src/ros_vrpn_client.cpp]: Please default to.... Hmm... "ros" here?

File: src/ros_vrpn_client.cpp [hunk 2]
```diff
@@ -293,29 +333,48 @@ int main(int argc, char* argv[])
...
+  }
+  else if (timestamping_system_string == "ros") {
+    timestamping_system = TimestampingSystem::kRosStamp;
+  }
+  else {
+    ROS_FATAL("ROS param timestamping_system should be either 'vicon' or 'ros'!");
+    return EXIT_FAILURE;
+  }
+
+
```
Comment (helenol) [inline src/ros_vrpn_client.cpp]: enter
enter
enter
enter




enter

**Comment by helenol** [comment]
Thanks a lot! LGTM. Maybe check with @burrimi what the default should be - current problem-prone behavior (vicon) or probably-not-quite-accurate behavior that has fewer issues (ros)

**Comment by alexmillane** [comment]
Talked to him in RL. Vicon it is.

**Pull request merged**